resources/startup.sh

```diff
@@ -27,9 +27,9 @@ if [[ ${loggingExitCode} -ne 0 ]]; then
   exitOnErrorWithMessage "ErrorRootLogLevelMapping" "ERROR: An error occurred during the root log level evaluation."
 fi
 
-echo "Waiting until ldap-mapper passed all health checks..."
-if ! doguctl healthy --wait --timeout 120 ldap-mapper; then
-  echo "timeout reached by waiting of ldap-mapper to get healthy"
+echo "Waiting until ldap passed all health checks..."
```

Comment:
This will not work with external LDAP or other directory services because the LDAP dogu will not be installed. This should be made a conditional.